repose-aggregator/components/services/datastore-service/datastore-service-impl/build.gradle

```diff
@@ -9,6 +9,8 @@ dependencies {
     compile "org.eclipse.jetty:jetty-servlet"
     compile "org.springframework:spring-beans"
     compile "net.sf.ehcache:ehcache-core"
+    compile "io.opentracing:opentracing-api"
+    compile "io.opentracing:opentracing-util"
```

Comment:
I glanced and didn't see anything, but is this dependency actually being used?

Reply:
Good call. These were moved into the `commons-utilities` project.